packages/@aws-cdk/aws-stepfunctions-tasks/test/emrcontainers/integ.start-job-run.ts

```diff
@@ -16,8 +16,24 @@ import { Classification, ReleaseLabel } from '../../lib/emrcontainers/start-job-
 const app = new cdk.App();
 const stack = new cdk.Stack(app, 'aws-stepfunctions-tasks-emr-containers-start-job-run-integ-test');
 
-const startJobRunJob = new EmrContainersStartJobRun(stack, 'EMR Containers Start Job Run', {
-  virtualClusterId: sfn.TaskInput.fromText('frv3my91yzjr1g0ha2h91obx3'),
+const createVirtualCluster = new cdk.CfnResource(stack, 'Create-a-virtual-cluster', {
+  type: 'AWS::EMRContainers::VirtualCluster',
+  properties: {
+    ContainerProvider: {
+      Id: 'test-eks',
+      Info: {
+        EksInfo: {
+          Namespace: 'spark2',
+        },
+      },
+      Type: 'EKS',
+    },
+    Name: 'Virtual-Cluster-Name',
+  },
+});
+
+const startJobRunJob = new EmrContainersStartJobRun(stack, 'Start a Job Run', {
+  virtualCluster: VirtualClusterInput.fromVirtualClusterId(cdk.Token.asString(createVirtualCluster.getAtt('Id'))),
   releaseLabel: ReleaseLabel.EMR_6_2_0,
   jobName: 'EMR-Containers-Job',
   jobDriver: {
```

Comment:
this integration test can be more minimal if you like (since the other test already checked the monitoring and app config). this just needs to check the custom resources